README.md

```diff
@@ -34,7 +34,8 @@
 
 ## 🎨 Features
 
-* **`sprocket lint`.** Lint Workflow Description Language files.
+* **`sprocket lint`** Lint Workflow Description Language files.
+* **`sprocket explain`** Explain lint warnings.
```

Comment:
Should this be `rules` instead of `warnings`?

Reply:
Nice catch, you are correct